app/src/ui/remove-repository/confirm-remove-repository.tsx

```diff
@@ -68,11 +68,9 @@ export class ConfirmRemoveRepository extends React.Component<
           <p>
             Are you sure you want to remove the repository "
             {this.props.repository.name}
-            "?
+            " from GitHub Desktop?
           </p>
           <p className="description">
-            The repository will be removed from GitHub Desktop:
```

Comment:
I  dont think theirs a need to delete this part  since this part is just a description   but adding the 'from GitHub Desktop'  to the heading can be  good idea to be more clear 

how its is currently:
![image](https://user-images.githubusercontent.com/17685135/123080596-0187a280-d415-11eb-8f49-a1ae86240852.png)